src/main/scala/chisel3/Driver.scala

```diff
@@ -241,10 +241,22 @@ object Driver extends BackendCompilationUtilities {
     val annosx = try {
       phases.foldLeft(annos)( (a, p) => p.transform(a) )
     } catch {
-      /* ChiselStage and FirrtlStage can throw StageError. Since Driver is not a StageMain, it cannot catch these. While
-       * Driver is deprecated and removed in 3.2.1+, the Driver catches all errors.
+      /* Catch all errors and show a fake stack trace. If the stack trace was minified elsewhere, e.g., in the Elaborate
+       * phase, then it will show up minified here. The Driver cannot fail for any reason. Instead, a
```

Comment:
what does "The Driver cannot fail for any reason" mean exactly? It is not allowed to throw any errors?

Reply:
Bad language on my part. It's saying that the Driver will catch all exceptions.

Specifically, if any Throwable that happens while the Driver is running (elaboration, conversion, etc.), then the Driver will always catch this, print out a fake stake trace on stdout, and then return an `ExecutionFailure` result. The Driver can fail and this will be visible to the user on stdout and they can check that an `ExecutionFailure` was returned to know that it failed.

Reply:
This is now irrelevant thanks to @sequencer's hard work on #1730.

Reply:
not resolve, when git clone new code, then run "sbt compile", happen a error 
"src/main/scala/chisel3/Driver.scala:54:12: object Driver is not a member of package firrtl
[error]     firrtl.Driver.execute(optionsManager) match {"

`trait BackendCompilationUtilities extends FirrtlBackendCompilationUtilities {
  /** Compile Chirrtl to Verilog by invoking Firrtl inside the same JVM
    *
    * @param prefix basename of the file
    * @param dir    directory where file lives
    * @return       true if compiler completed successfully
    */
  def compileFirrtlToVerilog(prefix: String, dir: File): Boolean = {
    val optionsManager = new ExecutionOptionsManager("chisel3") with HasChiselExecutionOptions with HasFirrtlOptions {
      commonOptions = CommonOptions(topName = prefix, targetDirName = dir.getAbsolutePath)
      firrtlOptions = FirrtlExecutionOptions(compilerName = "verilog")
    }

    firrtl.Driver.execute(optionsManager,) match {
      case _: FirrtlExecutionSuccess => true
      case _: FirrtlExecutionFailure => false
    }
  }
}
`
but Driver had be removed.  what can I do for running "sbt compile" successfully.